### PR TITLE
Update README according to the current toolchain

### PR DESCRIPTION
**Pull request by vdemeester**
Following https://github.com/docker/libcompose/pull/13#issuecomment-130565638, some updates on the README.md to be sync with the current toolchain.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -32,9 +32,29 @@ func main() {
...
 ## Building
 
-You need Docker and then run
-
-    ./script/build
+You need Docker and ``make`` and then run the ``binary`` target. This
+will create binary for all platform in the `bundles` folder. 
+
+```bash
+λ make binary
```
Comment (thaJeztah) [inline README.md]: nit, but could you change the prompt to a standard `$` for consistency? (here and in the other examples) :heart:

Comment (vdemeester) [inline README.md]: @thaJeztah lol, I like my `` sooo much though !! :blush:

Comment (thaJeztah) [inline README.md]: yeah, I saw the double "ticks" as well, but those don't show in the rendered page, so didn't want to bother you with that :-)

The prompt is something we try to be consistent with though.

Comment (vdemeester) [inline README.md]: @thaJeztah yeah I know :stuck_out_tongue_closed_eyes:, was just joking :smile_cat:, I'll put standard prompt as soon as I'm done eating :wink:

Comment (thaJeztah) [inline README.md]: No worries, and no hurry! Not home yet myself, so ;-)

**Comment by vdemeester** [comment]
I updated the name of the binaries (which are now `libcompose-cli`).

**Comment by thaJeztah** [comment]
LGTM thanks again, Vincent!

**Pull request merged**